Strings/Character escaping/character_escaping.py

```diff
@@ -1,4 +1,6 @@
-dont_worry = "Don't worry about apostrophes"
-# print(dont_worry)
-# print("\"Sweet\" is an ice-cream")
+dont_worry = "Don't worry about single quote if the string is in double quotes and vice versa"
```

Comment:
Don't worry about single quotes if the string is in double quotes, and vice versa.